pkgs/tools/security/yubihsm-shell/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, openssl
+, libusb
+, libedit
+, curl
+, gengetopt
+, pkg-config
+, pcsclite
+, help2man
+}:
+
+stdenv.mkDerivation rec {
+  pname = "yubihsm-shell";
+  version = "2.3.1";
+
+  src = fetchFromGitHub {
+    owner = "Yubico";
+    repo = "yubihsm-shell";
+    rev = version;
+    sha256 = "sha256-D0kXiwc+i6mKA4oHuHjgXUmLMsmY5o/VI+1aCWtNjC0=";
+  };
+
+  nativeBuildInputs = [ pkg-config cmake openssl help2man ];
+
+  buildInputs = [ libusb libedit curl gengetopt pcsclite ];
+
+  # help2man fails due to their cmake configuration
```

Comment:
Can you post the error log? Maybe someone can help.

Reply:
```
yubihsm-shell> [ 32%] Building manpage for yubihsm-auth
yubihsm-shell> help2man: can't get `--help' info from ./yubihsm-auth
yubihsm-shell> Try `--no-discard-stderr' if option outputs to stderr
yubihsm-shell> make[2]: *** [yubihsm-auth/CMakeFiles/yubihsm-auth-man.dir/build.make:73: yubihsm-auth/yubihsm-auth.1] Error 127
yubihsm-shell> make[1]: *** [CMakeFiles/Makefile2:707: yubihsm-auth/CMakeFiles/yubihsm-auth-man.dir/all] Error 2
yubihsm-shell> make[1]: *** Waiting for unfinished jobs....
yubihsm-shell> [ 32%] Built target yubihsm
yubihsm-shell> make: *** [Makefile:146: all] Error 2
```

Reply:
Did you try to replace `./` with `${CMAKE_CURRENT_BINARY_DIR}/` in https://github.com/Yubico/yubihsm-shell/blob/ac13b3b571a61bb4705a7bfbca1969bbbcd9e0ca/cmake/help2man.cmake#L8, then?

For the record, the following does work for me:

```
  postInstall = ''
    help2man "$out/bin/yubihsm-auth"
  '';
```

Reply:
I did, and it didn't seem to make a difference. My understanding is that `${CMAKE_CURRENT_BINARY_DIR}/` is the same expression as `./` in the end, so it's the same.

I like the postFixup solution, and if it's others do not consider it too much of a hack, I think it's worth merging.

Reply:
Thanks to what you said in Matrix, we have this working now. `-DCMAKE_SKIP_BUILD_RPATH=OFF` in the `cmakeFlags` was all it took. Thank you!